Updated README file with LinkedIn and Twitter links (#59)
wildfirepy/wildfirepy: README.md

------------

If you would like to get involved, start by joining the IRC chat room named `#wildfirepy` on [Riot](https://riot.im/app/#/room/!jWUOIxirCHymPQkpXb:matrix.org).

> Also if you wish to get notified for latest releases, follow our [LinkedIn](https://www.linkedin.com/company/wildfirepy/) and [Twitter](https://twitter.com/wildfirepy) handle as well.

Reviewer: ```suggestion
You may follow our [LinkedIn](https://www.linkedin.com/company/wildfirepy/) and [Twitter](https://twitter.com/wildfirepy) handle as well!
```
